db/sample_data_files/wild_collection/example_of_wild_collection_data.csv

```diff
@@ -1,14 +1,14 @@
 Tag,Collection_date,General Location,Precise Location,Collection_coodinates,Proximity to nearest neighbor (m),Collection method notes,Foot condition notes,Collection_depth_(m),Length,Weight,Gonad Score,Predicted Sex,Initial holding facility,Final holding facility & date of arrival,OTC treatment completion date
 Green_314,10/21/16,Los Angeles County,redacted,"1,1",>10,spatula,no cuts or abrasians reported,21.6,175,1242,1,M,AoP,BML; 12/13/2016,NA

```

Comment:
actually caught some invalid data :) (I hope)

Can someone confirm

Reply:
^ @corneliusellen @jcavena can either of you weigh in on if `redacted` should be invalid for `Precise Location`?  I don't know the project well enough to know.

Reply:
@wesleyeewong ```redacted``` is intentional. The precise lat/long location is confidential data.

Reply:
ok, but would end users also be uploading the lat/long as `redacted`? Or the actual lat/long?

Reply:
If end users will be uploading precise lat/longs then we should validate it, by faking the data here. If it'll always be `redacted`, then there's no reason to store that in the db.

Reply:
TBH I'm not 100%, but I think it will be uploaded as 'redacted'. I will check with the stakeholder. Thanks for bringing this to my attention.